tests/direct_beam_test.py

```diff
@@ -6,6 +6,93 @@
 from ess import polarization as pol
 
 
+def dummy_analyzer_spin() -> pol.CellSpinLog[pol.Analyzer]:
+    time = sc.array(dims=['time'], values=[0, 500], unit='s')
+    spin = sc.array(dims=['time'], values=[-1, 1], unit=None)
+    return pol.CellSpinLog[pol.Analyzer](sc.DataArray(spin, coords={'time': time}))
+
```

Comment:
This creates data as [DirectBeam, ++, Direct Beam, +-, DirectBeam, --, DirectBeam, -+]. I would change it to [DirectBeam, ++, -+, --, +-]x4 (or in principle x4 is not necessary). I have tried this in a follow up branch run-section-setup-all-spinstates-together